src/main/java/ca/corefacility/bioinformatics/irida/model/sequenceFile/SequencingObject.java

```diff
@@ -68,7 +69,7 @@
 	@NotAudited
 	private Set<QCEntry> qcEntries;
 
-	@ManyToMany(fetch = FetchType.LAZY, cascade = CascadeType.DETACH, mappedBy = "inputFiles")
+	@ManyToMany(fetch = FetchType.LAZY, cascade = CascadeType.DETACH, mappedBy = "inputFiles", targetEntity = AbstractAnalysisSubmission.class)
```

Comment:
I think `targetEntity` may need to be changed to `AnalysisSubmission` since you moved where the input files are stored down to `AnalysisSubmission`.

Reply:
This can be removed now since it's actually on `AnalysisSubmisison` again.  a3b384b7a2